packages/browser/src/core/events/index.ts

```diff
@@ -261,7 +261,7 @@ export class EventFactory {
       context,
       integrations: allIntegrations,
       ...overrides,
-      messageId: 'ajs-next-' + md5.hash(JSON.stringify(event) + uuid()),
+      messageId: 'ajs-next-' + md5.hash(JSON.stringify(event)) + uuid(),
```

Comment:
Maybe we just rip the band-aid off and do what we do in node.js?

https://github.com/segmentio/analytics-next/blob/master/packages/node/src/lib/get-message-id.ts

Reply:
updated the pr to reflect this change